README.txt

```diff
@@ -640,6 +640,14 @@ Use value of 2 for OS follow up.
 Disabled by default (enabling causing a slight performance
 degradation of ~50-100 nano sec per half round trip)
 
+VMA_RX_SW_CSUM
+This parameter enables/disables software checksum validation for received IP packets (TCP and UDP packets).
+When this parameter is enabled, software checksum validation will be calculated only if hardware checksum
```

Comment:
Sentence too long:
`When this parameter is enabled, software checksum validation will be calculated only if hardware checksum
 validation was not performed (in which case a performance degradation might occur).`

try something like:
`This parameter enables/disables software checksum validation for ingress TCP/UDP IP packets.
When this parameter is enabled, software checksum validation is calculated only if hardware offload checksum validation was not performed.
There is a slight performance degradation in cases which hardware offload failed to validation checksum and software calculation was used.
Note that disabling software calculation might cause corrupt packets to be processed by VMA and the application when hardware can't perform this action.
Read the VMA Release Notes for details HW adapters which support hardware offload checksum validation and which don't`